compiler/test/dotc/tests.scala

```diff
@@ -245,7 +245,7 @@ class tests extends CompilerTest {
   }
 
 
-  @Test def compileStdLib = compileList("compileStdLib", stdlibFiles, "-migration" :: "-Yno-inline" :: scala2mode)
+  @Test def compileStdLib = compileList("compileStdLib", stdlibFiles.map(_.replace("../scala-scala/src/library/", "")), "-migration" :: "-Yno-inline" :: scala2mode, Directory("../scala-scala/src/library/"))
   @Test def compileMixed = compileLine(
```

Comment:
This line looks a little strange, it seems the change can cancel each other? Otherwise, LGTM.